### PR TITLE
Use bom

### LINKED ISSUE #221: Upgrade to the latest google-cloud-logging library
We also should start using the Google Cloud Java BOM for managing library versions from Google Cloud Java.

Potential solution to #217.

### PR DESCRIPTION
**Pull request by ChengyuanZhao**
fixes #221

### CODE REVIEW

File: jetty9-base/pom.xml [hunk 1]
```diff
@@ -28,6 +28,18 @@
   <artifactId>jetty9-base</artifactId>
   <packaging>jar</packaging>
 
+  <dependencyManagement>
```
Comment (meltsufin) [inline jetty9-base/pom.xml]: Why not import it in the top-level `pom.xml`?

File: tests/test-war-smoke/pom.xml [hunk 2]
```diff
@@ -30,6 +30,19 @@
   <artifactId>test-war-smoke</artifactId>
   <packaging>war</packaging>
   <name>Jetty-Runtime :: Tests :: Smoke Tests (Servlet 3.1)</name>
+
+  <dependencyManagement>
```
Comment (meltsufin) [inline tests/test-war-smoke/pom.xml]: You shouldn't need to import the BOM here, if it's in the top-level `pom.xml`.

File: jetty9-base/pom.xml (hunk 1 above)
Comment (ChengyuanZhao) [reply jetty9-base/pom.xml]: Will do, new to maven so I didn't know one could do that.

**Comment by ChengyuanZhao** [comment]
@meltsufin changes made, PTAL

**Pull request merged**